pkg/controller/daemon/daemon_controller.go

```diff
@@ -687,6 +718,30 @@ func (dsc *DaemonSetsController) updateNode(logger klog.Logger, old, cur interfa
 	}
 }
 
+// getPodsFromCache returns the Pods that a given DS should manage.
+func (dsc *DaemonSetsController) getPodsFromCache(ds *apps.DaemonSet) ([]*v1.Pod, error) {
```

Comment:
getDaemonSetPods ?

Reply:
We have something similar below https://github.com/kubernetes/kubernetes/pull/130859/files#diff-073530752026c269afe84022a2d452456b7f7edcc2a71c05068ee601e3e14915R749

So, hence why I went with this I guess.

Reply:
The link does work for me somehow...